tests/test_matching_algorithms/test_pay_as_clear_matching_algorithm.py

```diff
@@ -42,8 +42,8 @@ def validate_matching(
             AssertionError
         """
         assert matching["selected_energy"] == matched_energy
-        assert matching["offer"]["id"] == offer_id
-        assert matching["bid"]["id"] == bid_id
+        assert matching["offers"][0]["id"] == offer_id
```

Comment:
I would suggest to also assert the length of the matching offers and bids (since we know that there should just be one of each, in this test)